pico/.gitignore

```diff
@@ -0,0 +1,16 @@
+README.md
```

Comment:
This file does not belong here. There is only one git ignore for the repo, in the root

Reply:
As I mentioned previously, I want to hide files not ready to merge down.

Reply:
Please remove this file. If you "un-add" your files from git, they will not be added to your commits and pushes.

Reply:
removed and pushed.